sites/includes/canonical.html

```diff
@@ -0,0 +1,10 @@
+{% comment %}
+Should only be called if page.altLangPage is set and a BasePath URL exist
+{% endcomment %}
+{%- if site.baseurl -%}
+<link rel="canonical" href="{{ page.url | prepend: site.baseurl }}" />
+<link rel="alternate" href="{{ page.altLangPage | prepend: site.baseurl | prepend: i18nText-altLang }}" hreflang="{{ i18nText-altLang }}" />
+{% elsif site.urlalt %}
+<link rel="canonical" href="{{ page.url | prepend: site.urlalt[ page.lang ] }}" />
+<link rel="alternate" href="{{ page.altLangPage | prepend: site.urlalt[ i18nText-altLang ] }}" hreflang="{{ i18nText-altLang }}" />
+{%- endif -%}
```

Comment:
Is this file valuable enough to not be in the "metadata.html" include file? Is it reusable outside the file "metadata.html" context of use?

Both file are currently pretty small, I am not sure currently it is valuable enough to be into 2 separate file. I am suggesting to only have it in one single file for now.

Reply:
I didn't quite know how to treat the canonical stuff in a Jekyll template. It assumes that sites would always be bilingual. If people are using the tools to mostly set up experiments, should they have the option to turn the canonical stuff off or customize it? If that's the case shouldn't it be separate from the mandatory metadata requirements?

Reply:
As discussed, the canonical link will be in another PR